src/dataset-builder/ConceptSetCreator.ts

```diff
@@ -1,15 +1,16 @@
 import _ from 'lodash/fp';
 import { h } from 'react-hyperscript-helpers';
 import { DomainConceptSet } from 'src/dataset-builder/DatasetBuilderUtils';
-import { DatasetModel, SnapshotBuilderConcept as Concept } from 'src/libs/ajax/DataRepo';
+import { Snapshot, SnapshotBuilderConcept as Concept, SnapshotBuilderSettings } from 'src/libs/ajax/DataRepo';
 
 import { ConceptSelector } from './ConceptSelector';
 import { homepageState, Updater } from './dataset-builder-types';
 import { OnStateChangeHandler } from './DatasetBuilder';
 
 export type ConceptSetCreatorProps = {
   readonly onStateChange: OnStateChangeHandler;
-  readonly dataset: DatasetModel;
+  readonly snapshot: Snapshot;
```

Comment:
Could this be `snapshotId` instead of `snapshot`? I think only the ID is needed here.